bin/ytdl.js

```diff
@@ -366,10 +369,16 @@ if (opts.infoJson) {
       var dataRead = 0;
       readStream.on('data', (data) => {
         dataRead += data.length;
-        if (dataRead === size) {
-          updateBar();
-        } else {
-          updateBarThrottled();
+
+        if (is_stdout_mutable) {
```

Comment:
Same here, I think this check could be placed a bit above, before the  `require` statements